<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">def change1(self, class_instance):
	class_instance.setupUI(self.page_2)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>800</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22222e</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QFrame" name="Header1">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>30</y>
        <width>911</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f66867</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="MenuText">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Myriad Pro</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>MENU</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Graph4Button">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>450</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #ffffff;
border: 2px solid #f66867;
	image: url(:/DownloadImage/Graphs/Graph4.jpg);
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="Graph3Frame">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>420</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#f66867;
border-radius: 15</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Graph3Text">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-10</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Myriad Pro</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255,255,255,0)</string>
       </property>
       <property name="text">
        <string>Graph 3</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="Graph1Frame">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f66867;
border-radius: 15</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Graph1Text">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-10</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Myriad Pro</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255,255,255,0)</string>
       </property>
       <property name="text">
        <string>Graph 1</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="Graph2Frame">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>180</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f66867;
border-radius: 15</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Graph2Text">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-10</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Myriad Pro</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255,255,255,0)</string>
       </property>
       <property name="text">
        <string>Graph 2</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Graph1Button">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #ffffff;
border: 2px solid #f66867;
	image: url(:/DownloadImage/Graphs/Graph1.jpg);
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="Graph2Button">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>210</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #ffffff;
border: 2px solid #f66867;
	image: url(:/DownloadImage/Graphs/Graph2.jpg);
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="Graph4Frame">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>420</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f66867;
border-radius: 15</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="Graph4Text">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>-10</y>
         <width>71</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Myriad Pro</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255,255,255,0)</string>
       </property>
       <property name="text">
        <string>Graph 4</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Graph3Button">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>450</y>
        <width>141</width>
        <height>131</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
background-color: #ffffff;
border: 2px solid #f66867;
	image: url(:/DownloadImage/Graphs/Graph3.jpg);
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>Header1</zorder>
     <zorder>Graph3Frame</zorder>
     <zorder>Graph1Frame</zorder>
     <zorder>Graph2Frame</zorder>
     <zorder>Graph1Button</zorder>
     <zorder>Graph2Button</zorder>
     <zorder>Graph4Frame</zorder>
     <zorder>Graph3Button</zorder>
     <zorder>Graph4Button</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QFrame" name="Header2">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>30</y>
        <width>911</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f66867</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="ResultText">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>20</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Myriad Pro</family>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>RESULT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="DownloadImage">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>64</y>
         <width>41</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/DownloadImage/Download.png);
background-color: rgba(250, 250, 250, 0);
border-color: rgba(250, 250, 250, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="ParameterButton">
       <property name="geometry">
        <rect>
         <x>770</x>
         <y>59</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0);
image: url(:/DownloadImage/Parameter.png);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="QuitButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>650</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f66867;

</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="QuitImage">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>644</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
image: url(:/DownloadImage/Quit.png)


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="PNGdownload">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>180</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Mangal</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f66867;
border-radius:20px
</string>
      </property>
      <property name="text">
       <string>PNG</string>
      </property>
     </widget>
     <widget class="QPushButton" name="JPEGdownload">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>230</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>246</red>
            <green>104</green>
            <blue>103</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Mangal</family>
        <pointsize>6</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f66867;
border-radius:20px
</string>
      </property>
      <property name="text">
       <string>JPEG</string>
      </property>
     </widget>
     <widget class="QFrame" name="Frame3_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>439</width>
        <height>419</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(43, 43, 58);
image: url(:/DownloadImage/graph.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="LoadingBar">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>670</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
border-radius: 7
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
